.gitignore

```diff
@@ -7,6 +7,7 @@
 *.mo
 *~
 *.swp
+dmojsite/
```

Comment:
Why ignore `dmojsite`?

I suppose `dmojsite` is the virtual environment, but I don't think anyone would create it in the same directory as the codebase.

Reply:
@hieplpvip yeah, let's just keep it local.